r-client/R/client.R

```diff
@@ -16,9 +15,9 @@
 #'                          in degrees. If not supplied, this will be calculated
 #'                          from consecutive observations}
 #' }
-#' @param hostName A string with the host running groundhog. By default, this 
```

Comment:
@jayqi: can you ignore white space changes?

Reply:
@Haoen-Cui if you click on the Gear icon when viewing the diff, you can hide whitespace changes in the UI.

Typically, we consider not-having-trailing-whitespace to be a good best practice, and we encourage people to turn on automatic-stripping in RStudio and other text editors.

Reply:
yeah it's totally fine to have your editor touch these other lines. I'm cool with it.